doc/cygnus-ngsi/installation_and_administration_guide/reliability.md

```diff
@@ -0,0 +1,50 @@
+
+I.e. a certain number of retries or time-to-live has to be configured for complete batches of NGSI events (a batch may contain from 1 to N NGSI events); additionally, the retry frequency must be configured, which is not constant and can be different for first retries, for instance.
+
+Batches for future retries are saved in memory independently of the type of the channel used. This is because NGSI events within batches are not re-inserted in the channel but maintained aggregated as a batch for performance purposes (the batch has not to be aggregated again). This means context information candidate for retry can be lost if Cygnus shuts down or crashes for any reason. In any case, this should not worry Cygnus users, since not persisted events would be lost without a retry mechanism as Cygnus' one.
+
+Dispatching policy regarding new incoming NGSI events and candidate for retries is currently hardcoded. It is based on always giving the priority to retries (if any), and then process new incoming events (if any). This could change in the future in the form of a configurable policy. In any case, the `batch_retry_intervals` parameter grants new incoming events are always processed in between candidates for retry, even if `batch_ttl` is set to `-1` (always retry).
+
+[Top](#top)
+
+## <a name="section2"></a>Traffic absorption peaks
+Because Cygnus inherits from Apache Flume, Cygnus agents of any type inherit an internal architecture based on the usage of channels communicating sources and sinks. Apart for achieving this communication, channels, which can be seen as buffers for incoming data, work as a mechanism for traffic absorption peaks.
+
+In the specific case of Cygnus for NGSI, this means the channel will be able to absorb peaks of notifications from a NGSI source (typically Orion Context Broker) while the configured sinks work hard in the persistence of data. Of course, timing regarding reception time and persistence time will be different, since sinks will work slower than sources (typical consuming-producer problem), but al least data will not be lost.
+
+A Cygnus administrator must carefully study the expected throughput for the whole architecture containing Cygnus and prevent high loads of NGSI notifications by configuring a channel capacity enough to absorb it.
+
+[Top](#top)
+
+## <a name="section3"></a>File channels
+File channels are another heritage from Apache Flume. This kind of channels are based on files, and unlike memory channels, information within this kind of channel lives between crashes.
+
+Using file channels is interesting if your notification rate is slower than Cygnus recovery time. Then, nothing will be lost. If not, using file channels or any other *volatile* channel will not imply any significant difference, since the amount of new incoming events lost while Cygnus is down could be much higher than the number of past events saved in files.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1434/commits/8e4a0bb5f267057216739a6a8d8622e662d11f4b